[2022-06-28T19:14:47.606727+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/"" at C:\Users\Pc\Desktop\Yeni klasör\my_project_directory\vendor\symfony\http-kernel\EventListener\RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/\" at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:186)"} []
[2022-06-28T19:14:47.667484+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:14:47.669861+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:14:47.911054+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"8cbdf4"},"request_uri":"http://127.0.0.1:8000/_wdt/8cbdf4","method":"GET"} []
[2022-06-28T19:18:14.883526+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","max":"1"},"request_uri":"http://127.0.0.1:8000/lucky/number/1","method":"GET"} []
[2022-06-28T19:18:14.887773+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:18:14.940987+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:18:14.943335+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:18:15.251170+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"6f18d5"},"request_uri":"http://127.0.0.1:8000/_wdt/6f18d5","method":"GET"} []
[2022-06-28T19:18:25.345427+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","max":"100"},"request_uri":"http://127.0.0.1:8000/lucky/number/100","method":"GET"} []
[2022-06-28T19:18:25.354216+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:18:25.431780+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:18:25.433849+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:18:25.656650+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"1dad8c"},"request_uri":"http://127.0.0.1:8000/_wdt/1dad8c","method":"GET"} []
[2022-06-28T19:23:48.921503+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-28T19:23:48.926093+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:23:48.950736+02:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\DependencyInjection\Exception\RuntimeException: "Could not resolve argument $max of "App\Controller\LuckyController::number()", maybe you forgot to register the controller as a service or missed tagging it with the "controller.service_arguments"?" at C:\Users\Pc\Desktop\Yeni klasör\my_project_directory\vendor\symfony\http-kernel\Controller\ArgumentResolver\NotTaggedControllerValueResolver.php line 79 {"exception":"[object] (Symfony\\Component\\DependencyInjection\\Exception\\RuntimeException(code: 0): Could not resolve argument $max of \"App\\Controller\\LuckyController::number()\", maybe you forgot to register the controller as a service or missed tagging it with the \"controller.service_arguments\"? at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\symfony\\http-kernel\\Controller\\ArgumentResolver\\NotTaggedControllerValueResolver.php:79)"} []
[2022-06-28T19:23:49.033661+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:23:49.036098+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:23:49.037425+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:23:49.037522+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:23:49.396396+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"cad3da"},"request_uri":"http://127.0.0.1:8000/_wdt/cad3da","method":"GET"} []
[2022-06-28T19:23:50.642293+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/lucky/number/100"" at C:\Users\Pc\Desktop\Yeni klasör\my_project_directory\vendor\symfony\http-kernel\EventListener\RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/lucky/number/100\" at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/lucky/number/100/\". at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:74)"} []
[2022-06-28T19:23:50.779899+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:23:50.782115+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:23:50.783362+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:23:50.783459+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:23:51.050840+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"8869cb"},"request_uri":"http://127.0.0.1:8000/_wdt/8869cb","method":"GET"} []
[2022-06-28T19:24:17.719527+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-28T19:24:17.729177+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:24:17.761694+02:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\DependencyInjection\Exception\RuntimeException: "Could not resolve argument $max of "App\Controller\LuckyController::number()", maybe you forgot to register the controller as a service or missed tagging it with the "controller.service_arguments"?" at C:\Users\Pc\Desktop\Yeni klasör\my_project_directory\vendor\symfony\http-kernel\Controller\ArgumentResolver\NotTaggedControllerValueResolver.php line 79 {"exception":"[object] (Symfony\\Component\\DependencyInjection\\Exception\\RuntimeException(code: 0): Could not resolve argument $max of \"App\\Controller\\LuckyController::number()\", maybe you forgot to register the controller as a service or missed tagging it with the \"controller.service_arguments\"? at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\symfony\\http-kernel\\Controller\\ArgumentResolver\\NotTaggedControllerValueResolver.php:79)"} []
[2022-06-28T19:24:17.849489+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:24:17.851276+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:24:17.852450+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:24:17.852541+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:24:18.117754+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"cf270f"},"request_uri":"http://127.0.0.1:8000/_wdt/cf270f","method":"GET"} []
[2022-06-28T19:25:09.811224+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-28T19:25:09.814626+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:25:09.830169+02:00] request.CRITICAL: Uncaught PHP Exception ErrorException: "Warning: Undefined variable $max" at C:\Users\Pc\Desktop\Yeni klasör\my_project_directory\src\Controller\LuckyController.php line 15 {"exception":"[object] (ErrorException(code: 0): Warning: Undefined variable $max at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\src\\Controller\\LuckyController.php:15)"} []
[2022-06-28T19:25:09.887717+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:25:09.889979+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:25:09.891327+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:25:09.891435+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:25:10.206212+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"8a6b72"},"request_uri":"http://127.0.0.1:8000/_wdt/8a6b72","method":"GET"} []
[2022-06-28T19:25:23.249720+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-28T19:25:23.260205+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:25:23.339732+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:25:23.342012+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:25:23.617915+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"4560d2"},"request_uri":"http://127.0.0.1:8000/_wdt/4560d2","method":"GET"} []
[2022-06-28T19:25:27.413877+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-28T19:25:27.423694+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:25:27.500146+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:25:27.502433+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:25:27.716752+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"bcec8b"},"request_uri":"http://127.0.0.1:8000/_wdt/bcec8b","method":"GET"} []
[2022-06-28T19:25:28.185045+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-28T19:25:28.192913+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:25:28.261234+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:25:28.262874+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:25:28.464573+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9d6a87"},"request_uri":"http://127.0.0.1:8000/_wdt/9d6a87","method":"GET"} []
[2022-06-28T19:25:28.756382+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-28T19:25:28.763744+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:25:28.830623+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:25:28.832250+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:25:29.046571+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f0d415"},"request_uri":"http://127.0.0.1:8000/_wdt/f0d415","method":"GET"} []
[2022-06-28T19:25:29.332660+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-28T19:25:29.340722+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:25:29.407314+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:25:29.408913+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:25:29.615157+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"182236"},"request_uri":"http://127.0.0.1:8000/_wdt/182236","method":"GET"} []
[2022-06-28T19:25:29.898500+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-28T19:25:29.906610+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:25:29.973347+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:25:29.975000+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:25:30.173922+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"45ded4"},"request_uri":"http://127.0.0.1:8000/_wdt/45ded4","method":"GET"} []
[2022-06-28T19:25:30.450719+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-28T19:25:30.458053+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:25:30.523902+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:25:30.525501+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:25:30.724260+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"b547b1"},"request_uri":"http://127.0.0.1:8000/_wdt/b547b1","method":"GET"} []
[2022-06-28T19:26:42.809570+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/deneme","method":"GET"} []
[2022-06-28T19:26:42.813153+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:26:42.842684+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:26:42.844950+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:26:43.131550+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"239d90"},"request_uri":"http://127.0.0.1:8000/_wdt/239d90","method":"GET"} []
[2022-06-28T19:26:44.230740+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/deneme","method":"GET"} []
[2022-06-28T19:26:44.239352+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:26:44.317256+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:26:44.319588+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:26:44.547275+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"15eb0a"},"request_uri":"http://127.0.0.1:8000/_wdt/15eb0a","method":"GET"} []
[2022-06-28T19:26:44.908383+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/deneme","method":"GET"} []
[2022-06-28T19:26:44.916669+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:26:44.989065+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:26:44.990607+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:26:45.186005+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"17477b"},"request_uri":"http://127.0.0.1:8000/_wdt/17477b","method":"GET"} []
[2022-06-28T19:26:47.128909+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/deneme","method":"GET"} []
[2022-06-28T19:26:47.136640+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:26:47.206674+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:26:47.208393+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:26:47.429758+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"d1308b"},"request_uri":"http://127.0.0.1:8000/_wdt/d1308b","method":"GET"} []
[2022-06-28T19:28:30.520151+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","deger":"12"},"request_uri":"http://127.0.0.1:8000/deneme/12","method":"GET"} []
[2022-06-28T19:28:30.523675+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:28:30.555316+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:28:30.557653+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:28:30.836299+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"6b8411"},"request_uri":"http://127.0.0.1:8000/_wdt/6b8411","method":"GET"} []
[2022-06-28T19:28:31.898143+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","deger":"12"},"request_uri":"http://127.0.0.1:8000/deneme/12","method":"GET"} []
[2022-06-28T19:28:31.907253+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:28:31.984249+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:28:31.986399+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:28:32.219756+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a20926"},"request_uri":"http://127.0.0.1:8000/_wdt/a20926","method":"GET"} []
[2022-06-28T19:28:32.717694+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","deger":"12"},"request_uri":"http://127.0.0.1:8000/deneme/12","method":"GET"} []
[2022-06-28T19:28:32.725318+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:28:32.794575+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:28:32.796226+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:28:32.993699+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"47dadb"},"request_uri":"http://127.0.0.1:8000/_wdt/47dadb","method":"GET"} []
[2022-06-28T19:28:33.818094+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","deger":"12"},"request_uri":"http://127.0.0.1:8000/deneme/12","method":"GET"} []
[2022-06-28T19:28:33.826338+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:28:33.896028+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:28:33.897666+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:28:34.088655+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"bda2cb"},"request_uri":"http://127.0.0.1:8000/_wdt/bda2cb","method":"GET"} []
[2022-06-28T19:28:41.775450+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","deger":"12"},"request_uri":"http://127.0.0.1:8000/deneme/12","method":"GET"} []
[2022-06-28T19:28:41.786670+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:28:41.873343+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:28:41.875589+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:28:42.149234+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"19418a"},"request_uri":"http://127.0.0.1:8000/_wdt/19418a","method":"GET"} []
[2022-06-28T19:28:44.476468+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","deger":"1223"},"request_uri":"http://127.0.0.1:8000/deneme/1223","method":"GET"} []
[2022-06-28T19:28:44.486799+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:28:44.564297+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:28:44.566463+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:28:44.779948+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"ee75b8"},"request_uri":"http://127.0.0.1:8000/_wdt/ee75b8","method":"GET"} []
[2022-06-28T19:28:46.987784+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","deger":"122312"},"request_uri":"http://127.0.0.1:8000/deneme/122312","method":"GET"} []
[2022-06-28T19:28:46.995913+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:28:47.065672+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:28:47.067345+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:28:47.281121+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"e35973"},"request_uri":"http://127.0.0.1:8000/_wdt/e35973","method":"GET"} []
[2022-06-28T19:28:53.100257+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","deger":"15"},"request_uri":"http://127.0.0.1:8000/deneme/15","method":"GET"} []
[2022-06-28T19:28:53.107633+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:28:53.176885+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:28:53.178527+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:28:53.387636+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9538e1"},"request_uri":"http://127.0.0.1:8000/_wdt/9538e1","method":"GET"} []
[2022-06-28T19:30:00.667530+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","deger":"15"},"request_uri":"http://127.0.0.1:8000/deneme/15","method":"GET"} []
[2022-06-28T19:30:00.677573+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:30:00.682609+02:00] php.CRITICAL: Uncaught Error: Call to undefined method App\Controller\LuckyController::generateUrl() {"exception":"[object] (Error(code: 0): Call to undefined method App\\Controller\\LuckyController::generateUrl() at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\src\\Controller\\LuckyController.php:15)"} []
[2022-06-28T19:30:00.710148+02:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\UndefinedMethodError: "Attempted to call an undefined method named "generateUrl" of class "App\Controller\LuckyController"." at C:\Users\Pc\Desktop\Yeni klasör\my_project_directory\src\Controller\LuckyController.php line 15 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\UndefinedMethodError(code: 0): Attempted to call an undefined method named \"generateUrl\" of class \"App\\Controller\\LuckyController\". at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\src\\Controller\\LuckyController.php:15)"} []
[2022-06-28T19:30:00.823374+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:30:00.825681+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:30:00.826904+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:30:00.827015+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:30:01.173068+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"cbe586"},"request_uri":"http://127.0.0.1:8000/_wdt/cbe586","method":"GET"} []
[2022-06-28T19:30:12.145086+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","deger":"15"},"request_uri":"http://127.0.0.1:8000/deneme/15","method":"GET"} []
[2022-06-28T19:30:12.156869+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:30:12.161482+02:00] php.CRITICAL: Uncaught Error: Call to undefined method App\Controller\LuckyController::generateUrl() {"exception":"[object] (Error(code: 0): Call to undefined method App\\Controller\\LuckyController::generateUrl() at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\src\\Controller\\LuckyController.php:15)"} []
[2022-06-28T19:30:12.191979+02:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\UndefinedMethodError: "Attempted to call an undefined method named "generateUrl" of class "App\Controller\LuckyController"." at C:\Users\Pc\Desktop\Yeni klasör\my_project_directory\src\Controller\LuckyController.php line 15 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\UndefinedMethodError(code: 0): Attempted to call an undefined method named \"generateUrl\" of class \"App\\Controller\\LuckyController\". at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\src\\Controller\\LuckyController.php:15)"} []
[2022-06-28T19:30:12.321519+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:30:12.324064+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:30:12.325368+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:30:12.325474+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:30:12.638518+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"4abec8"},"request_uri":"http://127.0.0.1:8000/_wdt/4abec8","method":"GET"} []
[2022-06-28T19:31:42.669633+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","deger":"15"},"request_uri":"http://127.0.0.1:8000/deneme/15","method":"GET"} []
[2022-06-28T19:31:42.673307+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:31:42.707306+02:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Routing\Exception\MissingMandatoryParametersException: "Some mandatory parameters are missing ("deger") to generate a URL for route "app_lucky_number"." at C:\Users\Pc\Desktop\Yeni klasör\my_project_directory\vendor\symfony\routing\Generator\UrlGenerator.php line 178 {"exception":"[object] (Symfony\\Component\\Routing\\Exception\\MissingMandatoryParametersException(code: 0): Some mandatory parameters are missing (\"deger\") to generate a URL for route \"app_lucky_number\". at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\symfony\\routing\\Generator\\UrlGenerator.php:178)"} []
[2022-06-28T19:31:42.759913+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:31:42.762264+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:31:42.763572+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:31:42.763683+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:31:43.114983+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"3a4b3a"},"request_uri":"http://127.0.0.1:8000/_wdt/3a4b3a","method":"GET"} []
[2022-06-28T19:32:20.403291+02:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number","deger":"12"},"request_uri":"http://127.0.0.1:8000/deneme/12","method":"GET"} []
[2022-06-28T19:32:20.413571+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:32:20.505512+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:32:20.507781+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:37:42.684854+02:00] console.CRITICAL: Error thrown while running command ""make:"". Message: "Command "make:" is ambiguous. Did you mean one of these?     make:auth                  Creates a Guard authenticator of different flavors     make:command               Creates a new console command class     make:controller            Creates a new controller class     make:crud                  Creates CRUD for Doctrine entity class     make:docker:database       Adds a database container to your docker-compose.yaml file     make:entity                Creates or updates a Doctrine entity class, and optionally an API Platform resource     make:fixtures              Creates a new class to load Doctrine fixtures     make:form                  Creates a new form class     make:message               Creates a new message and handler     make:messenger-middleware  Creates a new messenger middleware     make:registration-form     Creates a new registration form system     make:reset-password        Create controller, entity, and repositories for use with symfonycasts/reset-pass...     make:serializer:encoder    Creates a new serializer encoder class     make:serializer:normalizer Creates a new serializer normalizer class     make:subscriber            Creates a new event subscriber class     make:twig-extension        Creates a new Twig extension class     make:test                  Creates a new test class     make:validator             Creates a new validator and constraint class     make:voter                 Creates a new security voter class     make:user                  Creates a new security user class     make:migration             Creates a new migration based on database changes     make:stimulus-controller   Creates a new Stimulus controller." {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"make:\" is ambiguous.\nDid you mean one of these?\n    make:auth                  Creates a Guard authenticator of different flavors\n    make:command               Creates a new console command class\n    make:controller            Creates a new controller class\n    make:crud                  Creates CRUD for Doctrine entity class\n    make:docker:database       Adds a database container to your docker-compose.yaml file\n    make:entity                Creates or updates a Doctrine entity class, and optionally an API Platform resource\n    make:fixtures              Creates a new class to load Doctrine fixtures\n    make:form                  Creates a new form class\n    make:message               Creates a new message and handler\n    make:messenger-middleware  Creates a new messenger middleware\n    make:registration-form     Creates a new registration form system\n    make:reset-password        Create controller, entity, and repositories for use with symfonycasts/reset-pass...\n    make:serializer:encoder    Creates a new serializer encoder class\n    make:serializer:normalizer Creates a new serializer normalizer class\n    make:subscriber            Creates a new event subscriber class\n    make:twig-extension        Creates a new Twig extension class\n    make:test                  Creates a new test class\n    make:validator             Creates a new validator and constraint class\n    make:voter                 Creates a new security voter class\n    make:user                  Creates a new security user class\n    make:migration             Creates a new migration based on database changes\n    make:stimulus-controller   Creates a new Stimulus controller. at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\symfony\\console\\Application.php:759)","command":"\"make:\"","message":"Command \"make:\" is ambiguous.\nDid you mean one of these?\n    make:auth                  Creates a Guard authenticator of different flavors\n    make:command               Creates a new console command class\n    make:controller            Creates a new controller class\n    make:crud                  Creates CRUD for Doctrine entity class\n    make:docker:database       Adds a database container to your docker-compose.yaml file\n    make:entity                Creates or updates a Doctrine entity class, and optionally an API Platform resource\n    make:fixtures              Creates a new class to load Doctrine fixtures\n    make:form                  Creates a new form class\n    make:message               Creates a new message and handler\n    make:messenger-middleware  Creates a new messenger middleware\n    make:registration-form     Creates a new registration form system\n    make:reset-password        Create controller, entity, and repositories for use with symfonycasts/reset-pass...\n    make:serializer:encoder    Creates a new serializer encoder class\n    make:serializer:normalizer Creates a new serializer normalizer class\n    make:subscriber            Creates a new event subscriber class\n    make:twig-extension        Creates a new Twig extension class\n    make:test                  Creates a new test class\n    make:validator             Creates a new validator and constraint class\n    make:voter                 Creates a new security voter class\n    make:user                  Creates a new security user class\n    make:migration             Creates a new migration based on database changes\n    make:stimulus-controller   Creates a new Stimulus controller."} []
[2022-06-28T19:40:56.860027+02:00] request.INFO: Matched route "app_brand_new". {"route":"app_brand_new","route_parameters":{"_route":"app_brand_new","_controller":"App\\Controller\\BrandNewController::index"},"request_uri":"http://127.0.0.1:8000/brand/new","method":"GET"} []
[2022-06-28T19:40:56.863509+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:40:57.562262+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:40:57.564651+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:40:57.837009+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"aa3e2e"},"request_uri":"http://127.0.0.1:8000/_wdt/aa3e2e","method":"GET"} []
[2022-06-28T19:54:09.200190+02:00] request.INFO: Matched route "app_book". {"route":"app_book","route_parameters":{"_route":"app_book","_controller":"App\\Controller\\LuckyController::getBook","id":"3"},"request_uri":"http://127.0.0.1:8000/book/3","method":"GET"} []
[2022-06-28T19:54:09.203855+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:54:09.235362+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:54:09.237672+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:54:13.069592+02:00] request.INFO: Matched route "app_book". {"route":"app_book","route_parameters":{"_route":"app_book","_controller":"App\\Controller\\LuckyController::getBook","id":"2"},"request_uri":"http://127.0.0.1:8000/book/2","method":"GET"} []
[2022-06-28T19:54:13.079727+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:54:13.168083+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:54:13.170422+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:54:16.071209+02:00] request.INFO: Matched route "app_book". {"route":"app_book","route_parameters":{"_route":"app_book","_controller":"App\\Controller\\LuckyController::getBook","id":"1"},"request_uri":"http://127.0.0.1:8000/book/1","method":"GET"} []
[2022-06-28T19:54:16.079293+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:54:16.156168+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:54:16.157733+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:54:18.066272+02:00] request.INFO: Matched route "app_book". {"route":"app_book","route_parameters":{"_route":"app_book","_controller":"App\\Controller\\LuckyController::getBook","id":"10"},"request_uri":"http://127.0.0.1:8000/book/10","method":"GET"} []
[2022-06-28T19:54:18.074643+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:54:18.146892+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:54:18.148496+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:54:20.902760+02:00] request.INFO: Matched route "app_book". {"route":"app_book","route_parameters":{"_route":"app_book","_controller":"App\\Controller\\LuckyController::getBook","id":"0"},"request_uri":"http://127.0.0.1:8000/book/0","method":"GET"} []
[2022-06-28T19:54:20.910687+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:54:20.998360+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:54:21.000579+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:54:55.407042+02:00] request.INFO: Matched route "app_book". {"route":"app_book","route_parameters":{"_route":"app_book","_controller":"App\\Controller\\LuckyController::getBook","id":"0"},"request_uri":"http://127.0.0.1:8000/book/0","method":"GET"} []
[2022-06-28T19:54:55.417426+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:54:55.500541+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:54:55.502959+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:54:57.197088+02:00] request.INFO: Matched route "app_book". {"route":"app_book","route_parameters":{"_route":"app_book","_controller":"App\\Controller\\LuckyController::getBook","id":"01"},"request_uri":"http://127.0.0.1:8000/book/01","method":"GET"} []
[2022-06-28T19:54:57.207355+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:54:57.301767+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:54:57.304057+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:54:59.374833+02:00] request.INFO: Matched route "app_book". {"route":"app_book","route_parameters":{"_route":"app_book","_controller":"App\\Controller\\LuckyController::getBook","id":"3213"},"request_uri":"http://127.0.0.1:8000/book/3213","method":"GET"} []
[2022-06-28T19:54:59.382321+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:54:59.409137+02:00] request.CRITICAL: Uncaught PHP Exception ErrorException: "Warning: Undefined array key 3213" at C:\Users\Pc\Desktop\Yeni klasör\my_project_directory\src\Controller\LuckyController.php line 36 {"exception":"[object] (ErrorException(code: 0): Warning: Undefined array key 3213 at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\src\\Controller\\LuckyController.php:36)"} []
[2022-06-28T19:54:59.501306+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:54:59.503065+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:54:59.504328+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:54:59.504436+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:54:59.770976+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9cab41"},"request_uri":"http://127.0.0.1:8000/_wdt/9cab41","method":"GET"} []
[2022-06-28T19:55:03.591708+02:00] request.INFO: Matched route "app_book". {"route":"app_book","route_parameters":{"_route":"app_book","_controller":"App\\Controller\\LuckyController::getBook","id":"3"},"request_uri":"http://127.0.0.1:8000/book/3","method":"GET"} []
[2022-06-28T19:55:03.600062+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:55:03.627662+02:00] request.CRITICAL: Uncaught PHP Exception ErrorException: "Warning: Undefined array key 3" at C:\Users\Pc\Desktop\Yeni klasör\my_project_directory\src\Controller\LuckyController.php line 36 {"exception":"[object] (ErrorException(code: 0): Warning: Undefined array key 3 at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\src\\Controller\\LuckyController.php:36)"} []
[2022-06-28T19:55:03.710957+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:55:03.712629+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:55:03.713829+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:55:03.713918+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:55:03.995863+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"d1dd48"},"request_uri":"http://127.0.0.1:8000/_wdt/d1dd48","method":"GET"} []
[2022-06-28T19:55:51.212239+02:00] request.INFO: Matched route "app_book". {"route":"app_book","route_parameters":{"_route":"app_book","_controller":"App\\Controller\\LuckyController::getBook","id":"3"},"request_uri":"http://127.0.0.1:8000/book/3","method":"GET"} []
[2022-06-28T19:55:51.223080+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:55:51.308139+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:55:51.310461+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:55:57.610196+02:00] request.INFO: Matched route "app_book". {"route":"app_book","route_parameters":{"_route":"app_book","_controller":"App\\Controller\\LuckyController::getBook","id":"2"},"request_uri":"http://127.0.0.1:8000/book/2","method":"GET"} []
[2022-06-28T19:55:57.620048+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:55:57.711889+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:55:57.714327+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:57:40.745912+02:00] request.INFO: Matched route "app_book". {"route":"app_book","route_parameters":{"_route":"app_book","_controller":"App\\Controller\\LuckyController::getBook","id":"2"},"request_uri":"http://127.0.0.1:8000/books/2","method":"GET"} []
[2022-06-28T19:57:40.749642+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:57:40.780505+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:57:40.782787+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:57:47.650035+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/book"" at C:\Users\Pc\Desktop\Yeni klasör\my_project_directory\vendor\symfony\http-kernel\EventListener\RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/book\" at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/book/\". at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:74)"} []
[2022-06-28T19:57:47.793370+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:57:47.795561+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:57:47.796803+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:57:47.796972+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T19:57:48.076730+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"cd2f78"},"request_uri":"http://127.0.0.1:8000/_wdt/cd2f78","method":"GET"} []
[2022-06-28T19:57:52.209383+02:00] request.INFO: Matched route "app_books". {"route":"app_books","route_parameters":{"_route":"app_books","_controller":"App\\Controller\\LuckyController::getBooks"},"request_uri":"http://127.0.0.1:8000/books","method":"GET"} []
[2022-06-28T19:57:52.219661+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T19:57:52.298308+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T19:57:52.300174+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T20:17:40.634337+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/hello"" at C:\Users\Pc\Desktop\Yeni klasör\my_project_directory\vendor\symfony\http-kernel\EventListener\RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/hello\" at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\symfony\\http-kernel\\EventListener\\RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/hello/\". at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\symfony\\routing\\Matcher\\Dumper\\CompiledUrlMatcherTrait.php:74)"} []
[2022-06-28T20:17:40.729324+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T20:17:40.732154+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T20:17:40.733486+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T20:17:40.733612+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T20:17:41.219558+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"5eed87"},"request_uri":"http://127.0.0.1:8000/_wdt/5eed87","method":"GET"} []
[2022-06-28T20:42:00.297257+02:00] request.INFO: Matched route "hello". {"route":"hello","route_parameters":{"_route":"hello","_controller":"App\\Umut\\TestBundle\\Controller\\MerhabaBundleController::index"},"request_uri":"http://127.0.0.1:8000/hello","method":"GET"} []
[2022-06-28T20:42:00.301279+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T20:42:00.332600+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T20:42:00.334998+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
[2022-06-28T20:43:07.288250+02:00] request.INFO: Matched route "hello". {"route":"hello","route_parameters":{"_route":"hello","_controller":"App\\Umut\\TestBundle\\Controller\\MerhabaBundleController::index"},"request_uri":"http://127.0.0.1:8000/hello","method":"GET"} []
[2022-06-28T20:43:07.291856+02:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-28T20:43:07.327316+02:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-28T20:43:07.330043+02:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\API\\PostgreSQL\\ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at C:\\Users\\Pc\\Desktop\\Yeni klasör\\my_project_directory\\vendor\\doctrine\\dbal\\src\\Driver\\PDO\\PgSQL\\Driver.php:28)"} []
